CaseBonita/Services/PlaylistUploader/Handlers/Spotify.py

```diff
@@ -1,26 +1,77 @@
+import json
+import webbrowser
+
+import requests
 import spotipy as sp
 from spotipy.oauth2 import SpotifyClientCredentials
 
 from CaseBonita.Services.PlaylistUploader.Handlers.Base import BaseAPIUploader
-from .consts import CLIENT_ID, CLIENT_SECRET
+from .consts import CLIENT_ID, CLIENT_SECRET, SPOTIFY_USER_ACCESS
+from .utils import insert_id_to_url
```

Comment:
Why .utils and .consts?
That doesn't follow our conventions.
There's a Utils folder located under `CaseBonita/Utils`.
Try to create a filename that's indicative to the functionality you propose to add.
If you think the util here isn't relevant for the entire project, follow the pattern of creating a Utils folder under your service.

Reply:
I agree, they belong in a project level dir, will edit.